build.sbt

```diff
@@ -50,17 +50,6 @@ scalaVersion := "2.12.19"
 // keep this as low as possible to avoid running into binary incompatibility such as https://github.com/sbt/sbt/issues/5049
 pluginCrossBuild / sbtVersion := "1.3.1"
 
-scriptedSbt := {
-  val jdk = System.getProperty("java.specification.version").toDouble
-
-  if (jdk >= 21)
-    "1.9.0" // first release that supports JDK21
```

Comment:
this was actually required, but [untested](https://github.com/scalacenter/sbt-scalafix/pull/395#discussion_r1501714467)